packages/shared/notionable-hooks/src/use-wallet.ts

```diff
@@ -49,34 +50,34 @@ export function useWalletConnected() {
 
 export function useWalletAddress() {
   const {
-    globalState: { wallet },
-  } = useNotionalContext();
+    wallet: { userWallet },
+  } = useAppStore();
 
-  return wallet?.selectedAddress;
+  return userWallet?.selectedAddress;
 }
 
 export function useTruncatedAddress() {
   const {
-    globalState: { wallet },
-  } = useNotionalContext();
+    wallet: { userWallet },
+  } = useAppStore();
 
-  return wallet?.selectedAddress
-    ? truncateAddress(wallet?.selectedAddress)
+  return userWallet?.selectedAddress
+    ? truncateAddress(userWallet?.selectedAddress)
     : '';
 }
 
 export function useWalletConnectedNetwork() {
   const {
-    globalState: { wallet },
-  } = useNotionalContext();
-  return wallet?.selectedChain;
+    wallet: { userWallet },
+  } = useAppStore();
+  return userWallet?.selectedChain;
```

Comment:
The issue with this is that I'm not sure that onboard will always properly trigger an update inside the appstore for the user wallet. We may want to use some sort of async method to get the current selected chain inside the wallet.

Reply:
Not exactly sure what you're wanting here. We could always just use the onboard hooks here and return the chain directly

Reply:
The useEffect in the use-connect hook should keep the userWallet up to date how it has in the past though

Reply:
yeah use the onboard hooks instead